troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Introduction](#introduction) 
+
+* [Bootstrap VM related issues](#bootstrap-vm-related-issues)
+
+* [Node Related Issues](#node-related-issues)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+# Introduction
+ 
+**BM installation for OCP4:** While deploying BM version of Openshift there are some mandatory requirements which needs to be followed, otherwise it may fail. This guide attempts to cover some of the common scenarios of such events fixing which might help you get a clean cluster. 
+
+**Pre-requisites:** 
+
+First and foremost the BM installations follows a general architecture:
+* provisioner node/worker node(which can be converted after installation). 
```

Comment:
lets not label this provisioner/worker its confusing. Lets just leave it as provisioner node.

Reply:
Added the suggestion.